fonts.css

```diff
@@ -1,8 +1,7 @@
 @charset "UTF-8";
 
 /** Source Sans Pro, Source Serif Pro, Roboto Slab, Roboto Mono (regular only) */
-@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto+Slab:wght@300;400;700&family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&family=Source+Serif+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap');
-
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto+Slab:wght@300;400;700&family=Source+Sans+3:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Serif+4:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&display=swap');
```

Comment:
These font imports are only used for the preview site here; I replaced our sans and serif fonts, plus got rid of the 300 font weight since we never use them in the preview and won't (we don't even use that for our projects even).

Reply:
https://github.com/SU-SWS/saa_alumni/blob/dev/src/styles/global.css#L1

We totally can and do use them in projects

Reply:
@sherakama You're right we need to update the font import in the index.css file. I only updated it here in this font.css file 😬 Good catch.